src/input/AxisConfigWidget.cc

```diff
@@ -51,11 +51,24 @@ void AxisConfigWidget::AxesChanged(int nr, double val) const{
 	if(progressBar==nullptr) return;
 	progressBar->setValue(value);
 	progressBar->setFormat(s);
+	
+	QDoubleSpinBox* deadzone = this->findChild<QDoubleSpinBox *>(QString::fromStdString("doubleSpinBoxDeadzone"+number));
```

Comment:
Tiny suggestion: You can concatenate numbers directly to a QString: `QString("doubleSpinBoxDeadzone") + nr`, without going through a std::string.

Reply:
that makes things simpler - thank you